client/jetpack-connect/main.jsx

```diff
@@ -119,12 +119,12 @@ export class JetpackConnectMain extends Component {
 			from = 'ad';
 		}
 
-		const ctaProps = this.getCtaTrackingDetails();
+		const { cta_id, cta_from } = this.getCtaTrackingDetails();
 
 		this.props.recordTracksEvent( 'calypso_jpc_url_view', {
 			jpc_from: from,
-			cta_id: ctaProps.cta_id,
-			cta_from: ctaProps.cta_from,
+			cta_id: cta_id,
+			cta_from: cta_from,
```

Comment:
with es6, you don't need the duplication:
```js
{
  something: something
}
```

is the same as

```js
{
  something
}
```

so you can shorten this further to:
```js
{
  jpc_from: from,
  cta_id,
  cta_from,
}
```

Reply:
Sorry, unintentional!